splitio/storage/adapters/redis.py

```diff
@@ -297,6 +312,54 @@ def pipeline(self):
         except RedisError as exc:
             raise RedisAdapterException('Error executing ttl operation') from exc
 
+    def record_init(self, *values):
+        try:
+            host_name, host_ip = self._get_host_info()
+            return self.hset(TELEMETRY_CONFIG_KEY, 'python-' + __version__ + '/' + host_name+ '/' + host_ip, str(*values))
+        except RedisError as exc:
+            raise RedisAdapterException('Error pushing telemetry config operation') from exc
+
+    def _get_host_info(self):
+        host_name = 'Unknown'
+        host_ip = 'Unknown'
+        try:
+            host_ip = _get_ip()
+            host_name = socket.gethostname()
+        except:
+            _LOGGER.debug("Could not get hostname or ip")
+            pass
+        return host_name, host_ip
+
+    def record_stats(self, values):
```

Comment:
Redis won't record stats periodically, you will record latencies and exceptions at the time it happens